extensions/positron-python/src/client/interpreter/configuration/environmentTypeComparer.ts

```diff
@@ -49,17 +50,20 @@ export class EnvironmentTypeComparer implements IInterpreterComparer {
         this.workspaceFolderPath = this.interpreterHelper.getActiveWorkspaceUri(undefined)?.folderUri.fsPath ?? '';
     }
 
+    // --- Start Positron ---
     /**
      * Compare 2 Python environments, sorting them by assumed usefulness.
      * Return 0 if both environments are equal, -1 if a should be closer to the beginning of the list, or 1 if a comes after b.
      *
      * The comparison guidelines are:
      * 1. Local environments first (same path as the workspace root);
-     * 2. Global environments next (anything not local), with conda environments at a lower priority, and "base" being last;
-     * 3. Globally-installed interpreters (/usr/bin/python3, Microsoft Store).
+     * 2. Interpreters defined in the setting defaultInterpreterPath (Positron);
```

Comment:
it doesn't seem like the Python extension has the context of a registered affiliated runtime, but we can sort the default interpreter in a reasonable way here